### PR TITLE
Mvn publish GitHub action

### PR DESCRIPTION
**Pull request by elroykanye**
In this, I followed one of the current issues to add my email under the developers' section in pom.xml.
Also, two new files were included:

- the test class for the ConvertService
- a class Constants to contain project wide constants accessible to all other classes. In this, I declared the SRC and DEST folder variables.

Secondly, I have a suggestion concerning the github-action for deploying this to Maven Central. The suggested github action has been provided in the workflows dir without the **.yml** extension. It also requires following this guides:

1. https://central.sonatype.org/publish/publish-guide/
2. https://docs.github.com/en/actions/publishing-packages/publishing-java-packages-with-maven
3. https://github.com/chhh/sonatype-ossrh-parent/blob/master/publishing-to-maven-central.md
I think we could make some sense of these for that purpose.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,69 +1,71 @@
...
+            <version>0.22.0</version>
+            <scope>test</scope>
+        </dependency>
+
+
+        <dependency>
             <groupId>org.assertj</groupId>
             <artifactId>assertj-core</artifactId>
-            <version>3.9.1</version>
+            <version>3.22.0</version>
```
Comment (FanJups) [inline pom.xml:64]: @elroykanye , please what do you change the version of assert ?

Comment (elroykanye) [reply pom.xml:64]: I upgraded to 3.22
Any issue?

Comment (FanJups) [reply pom.xml:64]: OK no problem

**Comment by FanJups** [comment]
Please @elroykanye , I suggest to do a special PR for Github Action.  So, back on the previous state of .github folder. On this PR, we'll focus on the features you added.

 About maven central, let's focus first on the code, features, tests , javadocs ... then we'll publish on maven central.  The program works but we can do better to make it more professional.
 You can  open an issue with all the links you shared on this PR and open another PR  for Github Actions only.

**Comment by elroykanye** [comment]
> Please @elroykanye , I suggest to do a special PR for Github Action. So, back on the previous state of .github folder. On this PR, we'll focus on the features you added.
> 
> About maven central, let's focus first on the code, features, tests , javadocs ... then we'll publish on maven central. The program works but we can do better to make it more professional. You can open an issue with all the links you shared on this PR and open another PR for Github Actions only.

Okay, understood. Will the call we were supposed to have hold? If yes, when? I'd like to know all the features in vision for this so I can come up with an execution plan :)

**Comment by FanJups** [comment]
I will tell you tomorrow.

**Comment by FanJups** [comment]
I will open an issue to explain the steps

**Comment by FanJups** [comment]
> > Please @elroykanye , I suggest to do a special PR for Github Action. So, back on the previous state of .github folder. On this PR, we'll focus on the features you added.
> > About maven central, let's focus first on the code, features, tests , javadocs ... then we'll publish on maven central. The program works but we can do better to make it more professional. You can open an issue with all the links you shared on this PR and open another PR for Github Actions only.
> 
> Okay, understood. Will the call we were supposed to have hold? If yes, when? I'd like to know all the features in vision for this so I can come up with an execution plan :)

We already have the 3 features :
1. A method returning a String Object containing Js from a String Object containing Html as parameter.
2. A method generating a Js file from Html file as parameter.
3. CLI :  A method generating a Js file from Html file as argument

4. We'll add all the required tests, javadocs , refactor the code as much as possible. In order to better understand the Jsoup library, there are links in the README file.
5. Publish on Maven Central in order to get our own Maven / Gradle dependency.
6. Building the API with Spring Boot, we will 2 main endpoints related to 1. & 2. We will build the frontend with Angular or whatever...

I will open an issue describing all these steps. Now , you have all the steps.

Happy coding 😊

**Comment by elroykanye** [comment]
This is perfect! 💯 
Thanks much

**Pull request merged**